src/test/java/reposense/commits/CommitInfoAnalyzerTest.java

```diff
@@ -84,4 +85,17 @@ public void analyzeCommits_allAuthorMultipleCommitIgnoredNoDateRange_success() {
         Assert.assertEquals(commitResultsShort, commitResultsFull);
         Assert.assertEquals(commitInfos.size() - 2, commitResultsFull.size());
     }
+
+    @Test
+    public void analyzeCommits_yongAuthorNoCommitIgnoredNoDateRangeEmptyMessage_success() {
```

Comment:
method title too long, can just leave it as `analyzeCommits_noCommitMessage_success`